changeset:   7634:770d869450ea
parent:      7632:c35cdc528fff
user:        Mats Lidell <mats.lidell@lidells.se>
date:        Sat Mar 22 23:19:55 2025 +0100
summary:     Make xemacs.desktop Category consistent with GNU Emacs

diff -r c35cdc528fff -r 770d869450ea etc/xemacs.desktop
--- a/etc/xemacs.desktop	Thu Mar 20 09:40:30 2025 +0000
+++ b/etc/xemacs.desktop	Sat Mar 22 23:19:55 2025 +0100
@@ -30,5 +30,5 @@
 Icon=xemacs
 Type=Application
 Terminal=false
-Categories=Application;Utility;TextEditor;
+Categories=Development;TextEditor;
 StartupWMClass=XEmacs

